app/models/package_manager/go.rb

```diff
@@ -53,34 +53,54 @@ def self.update(name)
       # example: github.com/myexample/modulename/v2
       if (matches = name.match(VERSION_MODULE_REGEX))
```

Comment:
Might be nice to put this whole thing in another #update_base_version if name.match(VERSION_MODULE_REGEX) kinda thing